src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,32 @@
+<footer>
+  <div class="modMessage">
+    <div class="footer-img">
+      <img src="../../../img/footer.png" alt="">
+    </div>
+    <p>〒532-0011　大阪市淀川区西中島6丁目8-23テスト<br>
+      TEL.06-1234-5678 FAX.06-1234-5678　営業時間 / 9:00 - 22:00　定休日 / 不定休</p>
+      <div class="icon-set">
+        <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+        <a href="#"><i class="fa-brands fa-twitter"></i></a>
+        <a href="#"><i class="fa-brands fa-instagram"></i></a>
+        <a href="#"><i class="fa-solid fa-rss"></i></a>
+      </div>
+  </div>
+  <div class="footer-nav">
+    <div class="footer-nav-list">
+      <ul>
+        <li><a href="#">HOME</a></li>
+        <li><a href="#">CONCEPT</a></li>
+        <li><a href="#">SERVICE</a></li>
+        <li><a href="#">FLOOR</a></li>
+        <li><a href="#">INTERVIEW</a></li>
+        <li><a href="#">BLOG</a></li>
+        <li><a href="#">ACCESS</a></li>
+        <li><a href="#">DOWNLOAD</a></li>
+      </ul>
+    </div>
+  </div>
+  <div class="footer-bottom">
+    Copyright &copy;WordPressテーマ「Switch(tcd063)」 All Rights Reserved.
+  </div>
```

Comment:
```suggestion
  <small class="footer-bottom">
    Copyright &copy;WordPressテーマ「Switch(tcd063)」 All Rights Reserved.
  </small>
```
著作権表記にsmallタグを使うとわかってる感が出せます！

https://developer.mozilla.org/ja/docs/Web/HTML/Element/small